protocols/README.md

```diff
@@ -0,0 +1,9 @@
+# Protocols
+
+This package provides common ways for decoding bytes into data models (e.g. Zipkin Span). These data models can then be decoded into internal pdata representations. Similarly, pdata can be encoded into a data model which can then be encoded into bytes.
+
+[bytes](bytes): Common interfaces for encoding/decoding bytes from/to data models.
+
+[models](models): Common interfaces for encoding/decoding data models from/to pdata.
```

Comment:
What do you think about this:

```
[encoding](encoding): Common interfaces for encoding/decoding bytes from/to custom data models.

[translator](translator): Common interfaces for translating data models from/to pdata.
```

Not sure if `translator` or `translating` to be consistent with `encoding`

Reply:
I think translation might be more similar to encoding than translator. "Doing an encoding"/"Doing a translation" vs "Doing a translator". I'll do encoding/translation for now, see what you think.

Reply:
Maybe "encoder" and "translator" then to be consistent with "exporter", "consumer", etc. in the project?

Reply:
or "serializer" and "translator" I think these are the winners 🗡️